src/Color/Manipulate.elm

```diff
@@ -180,45 +181,43 @@ mixWithWeight c1 c2 weight =
             colorToTup c2
 
         w =
-            calculateWeight ( r1, g1, b1, a1 ) ( r2, g2, b2, a2 ) clampedWeight
+            calculateWeight a1 a2 clampedWeight
 
         rgbMixed =
             ( mixChannel w ( r1, r2 ), mixChannel w ( g1, g2 ), mixChannel w ( b1, b2 ) )
 
         alphaMixed =
-            a1 * clampedWeight + a2 * (1 - clampedWeight)
+            a1 * clampedWeight + a2 * (1.0 - clampedWeight)
```

Comment:
You're correct, it is not needed.  I'll update the PR tonight (US central time).